lib/buffy/throttle.ex

```diff
@@ -167,12 +167,9 @@ defmodule Buffy.Throttle do
         )
 
         case unquote(supervisor_module).start_child(unquote(supervisor_name), {__MODULE__, {key, args}}) do
-          {:ok, pid} ->
-            {:ok, pid}
-
-          :ignore ->
-            [{pid, _}] = unquote(registry_module).lookup(unquote(registry_name), {__MODULE__, key})
```

Comment:
would it be worth it to attempt a look, and return just `:ok` if we get nothing back?

We don't use this today, so not sure if it is worth though.

Reply:
I think i'd just add complexity to the type. Then the consumer needs to understand _why_ it can sometimes contain a pid and sometimes not, which kinda defeats the purpose of having all of this logic wrapped up. I think this is better because the user doesn't have to care much about how it works.

Reply:
Yeah, I don't think we want to be messing with the `pid` much anyway since we're deferring its management to Buffy